blog/_posts/2016-09-30-Thank-you-all-from-melbourne.md

```diff
@@ -0,0 +1,50 @@
+---
+title: The end of an amazing Summer(winter) Of Code!!
+layout: post
+created_at: Sun sept 30 2016
+permalink: blog/2016-09-30-amazing end of the winter
```

Comment:
in order for the permalink to work correctly, it needs hyphens (`-`) between each word. (i.e. `blog/2016-09-30-amazing-end-of-the-winter`)